bin/verify-links.sh

```diff
@@ -134,24 +134,46 @@ for file in ${mdFiles}; do
       continue
     fi
 
-    # Local file href - skip for now.
-    # TODO add support for checking these
-    if [ "${ref:0:1}" == "#" ]; then
-      ref=${ref:1}
-      if ! grep "^$ref$" ${tmp}anchors1 > /dev/null 2>&1 ; then
-        echo $file: Can\'t find anchor \'\#${ref}\' | tee -a ${tmp}3
+    # Local repo markdown link (i.e. ref contains a #) 
+    if [[ "${ref/\#}" != "${ref}" ]]; then
+      # Check to see if the link starts with a hash, if not we need to update the filepath
```

Comment:
Sorry about that. I was testing removing a block of code and failed to notice that wild tabs appeared when I put it back.